### ISSUE TITLE
Windows Registry 

### ISSUE THREAD

**Issue by Peep** [ddakebono/PalworldRcon#10]
https://github.com/ddakebono/PalworldRcon/blob/52e6c9f1a9ad66dbf55184de651e911f064cae2e/PalworldRcon/Logic/Settings.cs#L17

I don't think we should write/use the Windows registry for this kind of program -- and this does fail on some systems. Maybe writing into %userprofile%/AppData would be superior?

I can follow up with a pull request if I get some time, just wanted to mention this.

**Comment by MitoG** [comment]
yeah, writting to the registry, especially writing a clear text password seems like a really bad idea
